### ISSUE TITLE
2k, 4k support

### ISSUE THREAD

**Issue by niekdomi** [elkrien/catppuccin-grub#28]
My laptop has a resolution of 2560x1600 therefore the UI of the grub theme is to small and the text looks pixelated.

and yes i've set `GRUB_GFXMODE=...` as mentioned in the FAQ

**Comment by isabelincorp** [comment]
Just to double check, you rebuilt with ```sudo grub-mkconfig -o /boot/grub/grub.cfg``` after making the change correct?

**Comment by domi413** [comment]
Yes, I just tried it again. Still looks small and pixelated

**Comment by niekdomi** [comment]
Yes, I did rebuild the config after making the changes. Considering the recurring issues with `GRUB_GFXMODE` not functioning as expected, I believe creating an install script like the one used in [this theme](https://github.com/vinceliuice/grub2-themes) would be beneficial. Such a script could also handle display resolution adjustments seamlessly.

**Comment by bonswouar** [comment]
I also have a high-resolution display (`2880x1800`), but I believe the resolution is as expected (even without `GRUB_GFXMODE`).
The only "problem" is that the UI is not "upscaled" _(not the right word though - it's very small)_

An install script could indeed improve things! On the example above the `item_height` is 2.2 times bigger for 4k than 1080p, plus they use icons of different sizes
Easier solution than an install script would be to provide few different configs and let the user chose the right one, although I'm not sure it's ideal for uncommon resolutions

**Comment by kalzEOS** [comment]
Here to bump this up. I have 4k monitors and the text is too small.
